src/Adapter/Contact/QueryHandler/GetContactForEditingHandler.php

```diff
@@ -78,7 +88,7 @@ public function handle(GetContactForEditing $query)
                 $query->getContactId()->getValue(),
                 $contact->name,
                 $contact->email,
-                $contact->customer_service,
+                BoolParser::castToBool($contact->customer_service),
```

Comment:
@rokaszygmantas @matks created BoolParser to handle cases like: "0", "false" returned from legacy classes/request handlers instead of boolean false;

Reply:
This seems really overkill, doesn't a simple cast work? `(bool) $contact->customer_service` The only use case that the class can handle is if the string is equal to `'false'`  But I don't think the legacy class ever return this and expect it to be considered false
As for the behat values then we should simply use 0

Reply:
Yes, that's true, PrimitiveUtils class has 

```
    /**
     * @param string $element
     *
     * @return bool
     */
    public static function castStringBooleanIntoBoolean($element)
    {
        if ($element === 'false') {
            return false;
        }

        return boolval($element);
    }
```